composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
+        "php": ">=7.0.29",
```

Comment:
We cannot Downgrade here

Please, can you instead add a test case for this?
Then we can merge.

Reply:
set php version to 7.1